### ISSUE TITLE
Ensure the optional repo (rhel-*-server-optional-rpms) is present while installation.

### ISSUE THREAD

**Issue by tenstormavi** [distributed-system-analysis/pbench#863]

**Comment by ndokos** [comment]
That repo provides some of the perl dependencies of pbench on RHEL7, but I don't know what you mean by "ensuring". I think we can add something to the docs to say "if installation of Perl-Switch fails, and you are on RHEL7, please make sure that you have the rhel-7-server-optional-rpms repo installed" and provide a pointer to a how-to for installing the repo. Is this what you mean or are you suggesting code changes?

**Comment by tenstormavi** [comment]
@ndokos Yes, I meant the same if it fails for an RHEL user they should have a pointer for how to deal with it.
